docs/containertools/README.md

```diff
@@ -0,0 +1,108 @@
+# Container Tools Guide 
+
+## Introduction
+
+The purpose of this guide is to list a number of related Open-source projects that are available
+on [GitHub.com](https://github.com) that operate on
+[Open Container Initiative](https://www.opencontainers.org/) (OCI) images and containers.  This
+guide will give a high level explanation of the related container tools and will explain a bit
+on how they interact amongst each other.
+
+The tools are:  
+
+* [Buildah](https://github.com/projectatomic/buildah)
+* [CRI-0](https://github.com/kubernetes-sigs/cri-o)
```

Comment:
I am not sure but isn't it supposed to be `CRI-O` (vs `CRI-0` with a zero)? :smiley:

Reply:
I think it was an 'oh' instead of a 'zero', but you're right, it looks more zero like in the review panel.  Regardless I've just deleted the two instances of it and carefully inserted a captial "oh" in both spots.